src/Modules/Authontication/Components/ChangePass/ChangePass.tsx

```diff
@@ -1,8 +1,170 @@
-import React from 'react'
+import { useState } from "react";
+import { useForm } from "react-hook-form";
+import { PasswordValidation } from "../../../../constans/VALIDATIONS";
+import axios from "axios";
+import { AuthorizedToken, USERS_URLs } from "../../../../constans/END_POINTS";
+import { toast } from "react-toastify";
 
 export default function ChangePass() {
+  const [showOldPassword, setShowOldPassword] = useState(false);
+  const [showPassword, setShowPassword] = useState(false);
+  const [showConfirmPassword, setShowConfirmPassword] = useState(false);
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+    getValues,
+  } = useForm({ mode: "onBlur" });
+
+  const toggleVisibility = (setterFunction: any) => {
```

Comment:
We should avoid using any, and provide the correct types when using typescript. This will allow us to detect any potential errors while working in our app.

Reply:
what to do if not sure about the used type of data? use console.log(typeof(data)) right?

Reply:
We can get the setter function type if we hover over it in the useState or the setter function itself.
![image](https://github.com/user-attachments/assets/ddb38ca6-0b77-4a1e-97e7-7e2607e89e89)
![image](https://github.com/user-attachments/assets/54781e3b-cd2a-41bc-9218-4f1d51ab2f34)